back/internal/usersvc/internal/userdb/qryuser.go

```diff
@@ -212,11 +319,9 @@ func (s *UserDB) findRoles(ctx cx, x *pb.FndRolesReq, y *pb.RolesResp) error {
 	}
 
 	err = s.db.QueryRow(
-		"SELECT COUNT(*) FROM role WHERE role_id = ? OR role_name = ? LIMIT ? OFFSET ?",
+		"SELECT COUNT(*) FROM role WHERE role_id = ? OR role_name = ?",
```

Comment:
I think this would need to be role_id IN xyz OR role_name IN abc, but I'm losing sight a bit of what's what here.

Reply:
Changes have been made with latest commit